README.md

```diff
@@ -552,12 +675,16 @@ metadata:
         - version: 0.1
 ```
 
-The metadata defined in the `metadata` section will be added to all nodes created during the mapping process.
-In addition to the user defined metadata, a property field  `add_source_column_names_as` is also available.
-It allows to indicate the column name in which the data was found, as a property.
-For example, if the label of a node is extracted from the "indication" column, and you indicate `add_source_column_name_as: source_column`, the node will have a property: `source_column: indication`.
-This can be added to the metadata section as follows:
+The metadata defined in the `metadata` section will be added to all nodes
```

Comment:
All nodes and edges, except for the source columns, which are only added to nodes.